spyder/utils/qthelpers.py

```diff
@@ -502,7 +502,42 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
-    
+
+def create_control_layout(controls, main_widget):

```

Comment:
@timhoffm I disagree, it makes the code less readable, and we need to add extra logic to see if one thing is a layout or a completely different thing.